File: Appfrontend/src/app/models/peliculaSalaCineDTO.ts

export interface PeliculaSalaCineDTO {
  idPeliculaSala: number;
  idPelicula: number;
  idSalaCine: number;
  fechaPublicacion: string;
  fechaFin: string;
}

export interface PeliculaSalaCineCreateDTO {
  idPelicula: number;
  idSalaCine: number;
  fechaPublicacion: string;
  fechaFin: string;
}

export interface PeliculaSalaDetalleDTO {
  nombrePelicula: string;
  nombreSala: string;
  tipoSala: string;
  fechaPublicacion: string;
}

File: API-Cine/API-Cine/Models/DTOs/PeliculaSalaCineDTO.cs
namespace API_Cine.Models.DTOs
{
    public class PeliculaSalaCineDTO
    {
        public int IdPeliculaSala { get; set; }

        public int IdPelicula { get; set; }

        public int IdSalaCine { get; set; }

        public DateOnly FechaPublicacion { get; set; }

        public DateOnly FechaFin { get; set; }
    }

    public class PeliculaSalaCineCreateDTO
    {

        public int IdPelicula { get; set; }

        public int IdSalaCine { get; set; }

        public DateOnly FechaPublicacion { get; set; }

        public DateOnly FechaFin { get; set; }
    }

    public class PeliculaSalaDetalleDTO
    {
        public string NombrePelicula { get; set; }
        public string NombreSala { get; set; }
        public string TipoSala { get; set; }
        public DateOnly FechaPublicacion { get; set; }
    }


}
